layouts/partials/page-meta-links.html

```diff
@@ -10,13 +10,9 @@
   {{ warnf "Warning: use of `github_url` is deprecated. For details see https://www.docsy.dev/docs/adding-content/repository-links/#github_url-optional" -}}
   <a href="{{ $gh_url }}" target="_blank"><i class="fa-solid fa-pen-to-square fa-fw"></i> {{ T "post_edit_this" }}</a>
 {{ else if $gh_repo -}}
-  {{ $gh_repo_path := printf "%s/content/%s" $gh_branch $pathFormatted -}}
-  {{ if and ($gh_subdir) (.Site.Language.Lang) -}}
-    {{ $gh_repo_path = printf "%s/%s/content/%s/%s" $gh_branch $gh_subdir ($.Site.Language.Lang) $pathFormatted -}}
-  {{ else if .Site.Language.Lang -}}
-    {{ $gh_repo_path = printf "%s/content/%s/%s" $gh_branch ($.Site.Language.Lang) $pathFormatted -}}
-  {{ else if $gh_subdir -}}
-    {{ $gh_repo_path = printf "%s/%s/content/%s" $gh_branch $gh_subdir $pathFormatted -}}
+  {{ $gh_repo_path := printf "%s/content/%s" $gh_branch ( relLangURL $pathFormatted ) -}}
```

Comment:
So (for some reason the GH web UI won't let me comment on any lines outside your change) you need to do the following:

Change line 2 above from
 {{ $pathFormatted := replace .File.Path "\\" "/" -}}
to
{{ $pathFormatted := strings.TrimPrefix hugo.WorkingDir $.File.Filename -}}

Then change these lines to:
  {{ $gh_repo_path := printf "%s%s" $gh_branch $pathFormatted -}}
  {{ if $gh_subdir -}}
    {{ $gh_repo_path = printf "%s/%s%s" $gh_branch $gh_subdir $pathFormatted -}}

That works for both sites with language subdirectories and ones without!